mainhub/mainhub.ino

```diff
@@ -93,19 +117,63 @@ void update_game()
     cursor_index = 1; // Move to the second LED
     direction = 1; // Change the direction to right
   }
-  // Increment the cursor, but reset to zero if it is greater than the number of leds (Circle Path)
-  /*cursor_index = cursor_index + 1;
-  if (cursor_index >= NUM_LEDS)
-  {
-    cursor_index = 0;
-  }
-*/
+  //LEDs
+  //Light the cursor
+  leds[cursor_index] = rainbowColors[currentLevel];
+  FastLED.show();
+  // Now turn the LED off, then pause
+  leds[cursor_index] = CRGB::Black;
+  FastLED.show();
+
+   //Light the target range
+  leds[min_index] = CRGB::White;
+  FastLED.show();
+  leds[max_index] = CRGB::White;
+  FastLED.show();
+  
   Serial.print("Cursor Value: ");
   Serial.println(cursor_index);
 
 }
 
+void game_logic()
+//WIN CASE
+{   if (is_button_pressed) {
+    if (cursor_index >= min_index && cursor_index <= max_index) {
+        // Game won!
+      Serial.println("Game won!");
+      delay(1000);
+      // Update the game logic
+      cursor_index = 0;
+      wait_update = wait_update/2;
```

Comment:
Similar to the Colors array you could use an array of values which would allow you to precisely tune the enjoyment of the game.  (e.g. Halfing each time may get too difficult)